src/app/App.js

```diff
@@ -3,21 +3,63 @@ import Logo from './components/Logo';
 import createButtonRed from './components/ButtonRed';
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
 import createGameModeName from './components/GameModeName';
+import elemFactory from './utils/elementFactory';
```

Comment:
czemu skracamy nazwę?

Reply:
Without any special intention, but is it something bad?

Reply:
I wouldn't change it. I mean it's 'element' afterall. It's not a very long name also, so I don't see a need for shorteningn. It is understandable and self explanatory. Just like it's not arr, but array.

Reply:
Sure, like I said, I didn't do it on purpose and the edited App.js is just for example of use.